src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -119,55 +119,58 @@ var editCommentHandler = function(e) {
-    var deletedCommentRow = $(this).closest('li');
-    var formData = formObject.serialize();
-    var editForm = submitButton.parent().next().next().next();
-    var frCommentList = submitButton.closest('.comments');
-    var panelHeading = $(this).parents("[id^='panel_display-']").find('.panel-heading').first();
-    
     e.preventDefault();
-    
-    $.ajax({
-        type: 'POST',
-        url: submitButton.attr('href') + '?' + formData,
-        beforeSend: function() {
-            submitButton.html("<img src='/images/ajax-loader.gif'/>");
-        },
-        error: function() {
-            if (editForm.is(':visible')) {
-                setFormErrorMessage(editForm.find('div > a'), 'Failed to delete comment. Please try again.');
-            } else if (frCommentList.parent().find('div.delete_error_msg').length === 0) {
-                frCommentList.after('<div class="delete_error_msg alert alert-danger">'
-                                    + 'Failed to delete comment. Please try again.</div>');
-            }
-            submitButton.html('<span class="glyphicon glyphicon-trash glyphicon-primary"></span>');
-        },
-        success: function(data) {
-            if (data.isError) {
-                if (editForm.is(':visible')) {
-                    setFormErrorMessage(editForm.find('div > a'), data.errorMessage);
-                } else if (frCommentList.parent().find('div.delete_error_msg').length === 0) {
-                    frCommentList.after('<div class="delete_error_msg alert alert-danger">' + data.errorMessage + '</div>');
-                }
-                submitButton.html('<span class="glyphicon glyphicon-trash glyphicon-primary"></span>');
-            } else if (isInCommentsPage()) {
-                reloadFeedbackResponseComments(formObject, panelHeading);
-            } else {
-                var numberOfItemInFrCommentList = deletedCommentRow.parent().children('li');
-                if (numberOfItemInFrCommentList.length <= 2) {
-                    deletedCommentRow.parent().hide();
-                }
-                if (frCommentList.find('li').length <= 1) {
-                    frCommentList.hide();
+
+    BootboxWrapper.showModalConfirmation('Warning', 'Are you sure you want to remove this comment?', function() {
+        var formObject = submitButton.parent();
+        var deletedCommentRow = submitButton.closest('li');
+        var formData = formObject.serialize();
+        var frCommentList = submitButton.closest('.comments');
+        var panelHeading = submitButton.parents("[id^='panel_display-']").find('.panel-heading').first();
+
+        $.ajax({
+            type: 'POST',
+            url: submitButton.attr('href') + '?' + formData,
+            beforeSend: function() {
+                submitButton.html("<img src='/images/ajax-loader.gif'/>");
+            },
+            error: function() {
+                processDeleteError('Failed to delete comment. Please try again.', submitButton);
```

Comment:
Maybe name the function according to what it is actually doing, something like `showErrorMessage()`